test/runtests.jl

```diff
@@ -1,3 +1,12 @@
+# Working around https://github.com/JuliaLang/Pkg.jl/issues/2500 for CI
+if VERSION <= v"1.8-"
```

Comment:
Also Julia v1.8-beta3 picks up the wrong binaries for me on Fugaku

Reply:
oO It should not. Can you post a precise reproducer?

Reply:
I could reproduce also locally on my laptop with system OpenMPI.  I did `MPIPreferences.use_system_binary()` to find the system MPI, then restarted julia, ran the tests and https://github.com/JuliaParallel/MPI.jl/blob/690378f6ef85bffd3c8c5c22b31631fb9fc462a7/test/runtests.jl#L23 was still showing MPICH_jll was being used.  Could reproduce also with julia `master` from a few days ago (`Commit 7529dbd021 (2022-04-15 16:18 UTC)`)

Reply:
Where was your `LocalPreferences.toml` file located? in `test/` I couldn't reproduce your issue on `-beta3`

Reply:
In my global environment. I made this work by activating the test environment and setting the preferences here, as I wrote in https://github.com/JuliaParallel/MPI.jl/pull/564#issuecomment-1101296494.

Related to what you wrote in https://github.com/JuliaParallel/MPI.jl/pull/564#issuecomment-1101414532, I think it'd be better propagating the preferences from the environment where you launch the test, also because this directory isn't supposed to be writable.

Reply:
> Related to what you wrote in [#564 (comment)](https://github.com/JuliaParallel/MPI.jl/pull/564#issuecomment-1101414532), I think it'd be better propagating the preferences from the environment where you launch the test, also because this directory isn't supposed to be writable.

That's a good argument. Also see https://github.com/JuliaLang/Pkg.jl/pull/2920
cc: @staticfloat